GOVERNANCE.md

```diff
@@ -0,0 +1,194 @@
+
+Collaborators can communicate in private with the Core Team [on GitHub](https://github.com/eslint-community/collaborators/discussions).
+
+### Collaborator activities
+
+* Helping users and novice contributors
+* Contributing code and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Merging pull requests
+* Release plugins
+
+The Core Team can remove inactive Collaborators or provide them with
+_Past Collaborators_ status. Past Collaborators may request that the Core
+Team restore them to active status.
+
+## Projects
+
+There are two kinds of projects in ESLint Community: Primary and auxiliary projects.
+
+Primary projects are maintained by Core Team in collaboration with project Collaborators
+and will be ensured to be kept up to date at a best effort pace by the Core Team.
+
+Auxiliary projects are maintained by its project Collaborators and is kept up to date at
+whatever pace those Collaborators chose. The Core Team merely fascilitates nominations
+of new Collaborators to such projects, ensuring there's a path to keep it maintained,
+without making any promises.
+
+All project npm modules should be owned (and solely owned) by the `eslint-community-bot`
+user on npm. If the npm modules are owned by an existing organisation and using its prefix,
+then the `eslint-community-bot` should become the sole owner of that organisation.
+
+Additional assets such as domain names, web sites etc should also be in control of the
+Core Team and/or the ESLint TSC.
+
+If possible, a project should always be transfered and keep using its original npm module
+name and repository. If a new module name needs to be created, it should preferably be
+`@eslint-community/<the-old-name>`.
+
+Automatic releases of the modules should be set up by the Core Team using the organization
+secret in the GitHub organization.
+
+Auxiliary projects can can be nominated to become a primary project by its Collaborators,
+by opening a private team discussion [on GitHub](https://github.com/eslint-community/collaborators/discussions).
+
+If a primary project is no longer widely depended upon, or have technical reasons to no
+longer stay a primary project, the Core Team can decide to convert it into an auxiliary
+project in discussion with its Collaborators.
+
+## Project nominations
+
+Projects that are widely depended upon within the ESLint ecosystem may be nominated to
+become ESLint-Community projects by anyone by opening a public issue [on GitHub](https://github.com/eslint-community/governance).
+The project should fulfill a few criterias:
+
+The project should:
+
+* Preferably be: An ESLint plugin or formatter
+* Maybe be: A dependency used by an ESLint-Community project or the main ESLint repo
+* Not be: Primarily an ESLint config
+
+The project should also:
+
+* be widely depended upon throughout the community (eg. 3M downloads/week or similar)
+* have an OSI-approved license (preferably same as ESLint itself: MIT)
```

Comment:
Ideally Apache 2.0. The OpenJS Foundation now expects Apache 2.0 for all new projects.

Reply:
@nzakas Good 👍 I guess we should also note whether ESLint-Community projects are to be considered to be under the OpenJSF umbrella or not

Reply:
Tweaked the wording to be `preferably Apache 2.0, else MIT-style`

Reply:
They aren't under the OpenJS umbrella, but I think their logic is sound in requiring Apache 2.0 vs. MIT-style for new projects as the former contains a patent provision that companies generally like to see spelled out.